packages/@aws-cdk/aws-appmesh/test/test.gateway-route.ts

```diff
@@ -114,27 +113,6 @@ export = {
       test.done();
     },
 
-    'should throw an exception if you start an http prefix match not with a /'(test: Test) {
-      // GIVEN
-      const stack = new cdk.Stack();
-
-      const mesh = new appmesh.Mesh(stack, 'mesh', {
-        meshName: 'test-mesh',
-      });
-
-      const virtualService = new appmesh.VirtualService(stack, 'testVirtualService', {
-        virtualServiceProvider: appmesh.VirtualServiceProvider.none(mesh),
-      });
-      test.throws(() => appmesh.GatewayRouteSpec.http({
-        routeTarget: virtualService,
-        match: {
-          prefixPath: 'wrong',
-        },
-      }).bind(stack),
-      /Prefix Path must start with \'\/\', got: wrong/);
-      test.done();
-    },
```

Comment:
We actually do not enforce `prefix` match to start with `/` as of now and thus removing it.
There will be a backlog action item to add this validation but date has not been confirmed.

Reply:
@skinny85 - I would like to hear your opinion about removing this validation since if we are adding back this, it would a breaking change.

I think this concern is related to [this question](https://github.com/aws/aws-cdk/pull/15527#discussion_r669098268).

Reply:
Lastly, if we agree to remove this validation from CDK, similar validation is also removed from [`HttpRoutePathMatch`](https://github.com/aws/aws-cdk/pull/15527#discussion_r669098268).